mlflow/store/artifact/databricks_models_artifact_repo.py

```diff
@@ -122,8 +122,8 @@ def _get_signed_download_uri(self, path=None):
             json_response = json.loads(response.text)
         except ValueError:
             raise MlflowException(
-                "API request to get presigned uri to for file under path `%s` failed with"
-                " status code %s. Response body: %s" % (path, response.status_code, response.text)
+                f"API request to get presigned uri to for file under path `{path}` failed with"
+                f" status code {response.status_code,}. Response body: {response.text}"
```

Comment:
```suggestion
                f" status code {response.status_code}. Response body: {response.text}"
```

Reply:
thanks for the catch!

Reply:
@gabrielfu Looks like AI is capable of finding these mistakes: https://github.com/mlflow/mlflow/pull/8955/files#r1252496491

Reply:
Nice!

Reply:
btw `/ai` is a spell (only I can use because it's powered by my 💰) to chat with AI.